thirdparty/linuxbsd_headers/README.md

```diff
@@ -9,20 +9,21 @@ readability.
 
 - Upstream: https://www.alsa-project.org/
 - Version: 1.1.3-5
-- License: LPGL-2.1+
+- License: LGPL-2.1+
 
 Patches in the `patches` directory should be re-applied after updates.
 
+
 ## dbus
 
-- Upstream: https://www.freedesktop.org/wiki/software/dbus/
+- Upstream: https://www.freedesktop.org/wiki/Software/dbus/
 - Version: 1.12.2
-- License: AFL-2.1
+- License: GPL-2.1
```

Comment:
The files say `Academic Free License`, if they *are* under the GPL we can't use them AFAIK, see [here](https://docs.godotengine.org/en/stable/contributing/development/best_practices_for_engine_contributors.html#don-t-use-complex-canned-solutions-for-simple-problems)

Reply:
Sorry about that, fixed.